rest/sync_fn_test.go

```diff
@@ -603,19 +603,21 @@ func TestResync(t *testing.T) {
 			})
 			assert.NoError(t, err)
 
-			assert.Equal(t, testCase.expectedSyncFnRuns, int(rt.GetDatabase().DbStats.Database().SyncFunctionCount.Value()))
-
 			if !isDCPResync {
 				var queryName string
+				assert.Equal(t, testCase.expectedSyncFnRuns, int(rt.GetDatabase().DbStats.Database().SyncFunctionCount.Value()))
```

Comment:
These assertions are all dead - it is _always_ DCP resync unless `UseQueryBasedResyncManager` is set.